coordinator/src/services/translation/prescription/index.ts

```diff
@@ -5,6 +5,8 @@ import * as prescriptions from "../../../models/hl7-v3/hl7-v3-prescriptions"
 import {DaysSupply, PrescriptionPertinentInformation7, ReviewDate} from "../../../models/hl7-v3/hl7-v3-prescriptions"
 import * as fhir from "../../../models/fhir/fhir-resources"
 import {
+  ContentReferencePayload,
```

Comment:
just noticed we `import * as fhir` above so can just use `fhir.Content...` in place

Reply:
Done